data_model.md

```diff
@@ -262,6 +262,8 @@ permitted are as follows:
 -   *DC power* - megawatts
 -   *PDF probability* - %
 -   *CDF value*
+-   *Availability* - %
+-   *Curtailment* - megawatts
```

Comment:
Are we asking for the amount of power that is curtailed, or the power generated during curtailed periods?  Needs to be clarified somewhere.  We might want to use % as the unit here.  I don't know what the conventional use is, maybe ask Ted.

Reply:
The limit that is imposed during curtailed periods. Our experience is that curtailment is specified in MW and availability in %. I suspect that we're not going to make use of this in year 1 so it's probably ok if we don't get it right the first time. 

We should probably add descriptions for all of these variables. I can't recall if we had them in an earlier document or not. This seems like this is the right place for it. Am I forgetting another list? I don't want to duplicate things.

Reply:
After some discussion here, I created a [Glossary wiki](https://github.com/SolarArbiter/solarforecastarbiter-core/wiki/Glossary) in the core repository and linked to it below. I think that is the a relatively efficient way to move forward.